templates/partials/filters/date.html

```diff
@@ -1,5 +1,5 @@
-<span id="dateRangeDropdown" class="filter dropdown mm-dropdown">
-  <span id="date-filter" class="mm-button">
+<span id="dateRangeDropdown" class="filter dropdown mm-dropdown" ng-class="{ 'disabled': selectMode && selected.length }">
```

Comment:
That was my initial plan too, but adding the class to that specific element means we get some of the disabling for free from bootstrap. If we put it on the parent element we'd have to write more css to implement our own disabling.